builtins/methods/pystring/dialect_formatter_spec_integration_test.go

```diff
@@ -0,0 +1,276 @@
+//go:build integration
```

Comment:
Note that this integration test requires a python binary to verify `.format()`-results against.